src/qml/NavigatingDrawer.qml

```diff
@@ -0,0 +1,125 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import org.qfield 1.0
+import Theme 1.0
+
+Drawer {
```

Comment:
How does this Drawer interact with the dashboard (which is also a drawer)? Are they stacking properly?

Reply:
`InformationDrawer` is depend on `mapCanvasMap.isEnabled` there for when `dashboard` is open `InformationDrawer` is hidden and after closing `dashboard`, `InformationDrawer` reappears.

Reply:
@mohsenD98 , that's not a good variable to use here, we still want info drawer when e.g. we lock the screen. 

For now, let's just have both drawers show, and insure that the dashboard sits on top.